examples/features/effect-handlers.effekt.md

```diff
@@ -93,3 +97,65 @@ def genFibs(limit: Int): List[Int] / {} = {
 ```effekt:repl
 genFibs(15)
 ```
+
+## Singleton operation
+
+Often you want to declare an interface that is entirely defined by just one operation. In this case you can declare it as a singleton operation:
+
+```effekt:hide
+extern def time(): Int =
+  js "Date.now()"
+```
+
+```
+effect tell(): Int
+```
+
+The handler for this interface uses a slightly different more concise syntax like it is used in the literature:
+
+```
+def tellTime[A] { prog: () => A / tell }: A =
+  try { prog() }
+  with tell { () =>
+    resume(time())
+  }
+```
+
+```effekt:repl
+tellTime {
+  val start = do tell()
+  val end = do tell()
+  println("took " ++ show(end - start))
+}
+```
```

Comment:
Nitpick: this example feels very artificial, we're not even measuring anything here.
We have `genFibs` in the same file, so we can use it here, no?
``````suggestion
```effekt:repl
tellTime {
  val start = do tell()
  val fibs = genFibs(8)
  val end = do tell()
  println("took " ++ show(end - start))
}
```
``````